### ISSUE TITLE
[New Feature] Fiter by Date/Time/Last X Hours

### ISSUE THREAD

**Issue by bonkersGER** [devedse/DeveLanCacheUI_Frontend#12]
Would it be possible to add a filter in the frontend to see the data of e.g. a certain period.

Start Date / Time - End Date / Time
Last X hours

**Comment by devedse** [comment]
Yeah it should definitely be possible, my frontend skills aren't that great though so I could use some help there :)
